script/github-actions/select-cypress-tests.js

```diff
@@ -0,0 +1,110 @@
+const core = require('@actions/core');
```

Comment:
Do you think there'd be any value in adding unit tests for this script?

Reply:
Peter and I decided to created a bunch of tests in TestRail and manually test Test Selection. I think writing unit tests for that file would be very time consuming.